rasa/core/policies/ted_policy.py

```diff
@@ -1261,20 +1215,23 @@ def _encode_real_features_per_attribute(
             sequence_lengths += 1
             mask_text = tf.squeeze(self._compute_mask(sequence_lengths), axis=1)
 
-            attribute_features, _, _, _, _ = self._create_sequence(
-                tf_batch_data[attribute][SEQUENCE],
-                tf_batch_data[attribute][SENTENCE],
-                mask_sequence_text,
-                mask_text,
-                attribute,
-                sparse_dropout=self.config[SPARSE_INPUT_DROPOUT],
-                dense_dropout=self.config[DENSE_INPUT_DROPOUT],
+            attribute_features, _, _, _, _ = self._tf_layers[
+                f"sequence_layer.{attribute}"
+            ](
+                (
```

Comment:
I agree. However, this doesn't work with Keras because of the implementation [here](https://github.com/tensorflow/tensorflow/blob/d5727f7c236b0be4292bb8fc7929a40592b48f0b/tensorflow/python/keras/engine/base_layer.py#L3057). It doesn't support the `inputs` argument in a named fashion...

Reply:
You get something like:
```
sequence = self._tf_layers[f"sparse_dense.{SEQUENCE}"](
    /Users/sams/.miniconda/envs/rasaenv/lib/python3.7/site-packages/tensorflow/python/keras/engine/base_layer.py:916 __call__  **
        inputs, args, kwargs = self._split_out_first_arg(args, kwargs)
    /Users/sams/.miniconda/envs/rasaenv/lib/python3.7/site-packages/tensorflow/python/keras/engine/base_layer.py:2980 _split_out_first_arg
        'The first argument to `Layer.call` must always be passed.')

    ValueError: The first argument to `Layer.call` must always be passed.
```

Reply:
🙈